src/naive/internals.rs

```diff
@@ -420,8 +421,10 @@ impl Mdf {
         }
     }
 
+    // Exported for pyo3.
+    #[doc(hidden)]
     #[inline]
-    pub(super) fn month(&self) -> u32 {
+    pub fn month(&self) -> u32 {
```

Comment:
For this, how about we add a method on `NaiveDate` that yields a `Month` and add a method on `Month` that gives you a `u8`?

Reply:
`NaiveDate` already exposes a month function that returns u32.
```rust
    #[inline]
    fn year(&self) -> i32 {
        self.ymdf >> 13
    }
    #[inline]
    fn month(&self) -> u32 {
        self.mdf().month()
    }
    #[inline]
    fn day(&self) -> u32 {
        self.mdf().day()
    }
```